operator_ui/src/containers/JobRuns/Index.js

```diff
@@ -23,6 +23,8 @@ const styles = theme => ({
 
 const renderLatestRuns = (props, state, handleChangePage) => {
   const { jobSpecId, latestJobRuns, jobRunsCount, pageSize } = props
+  const pagePath = props.pagePath.replace(':jobSpecId', jobSpecId)
```

Comment:
Why not deconstruct pagePath above and use it immediately if we're already deconstructing it?

Reply:
You can't reassign a `const`